lib/markdown_generators.js

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env node
+
+const yaml = require("js-yaml");
+const markdown = require("markdown-builder");
+const title_case = require("title-case");
+const TurndownService = require("turndown");
+const tables = require("turndown-plugin-gfm").tables;
+const turndown_service = new TurndownService();
+turndown_service.use(tables);
+const helpers = require("./helpers");
+
+class markdown_generators {
+    static generate_course_home_front_matter(course_data) {
+        /*
+        Generate the front matter metadata for the course home page given course_data JSON
+        */
+        let front_matter = {};
+        front_matter["title"] = "Course Home";
+        front_matter["course_title"] = course_data["title"];
+        front_matter["course_image_url"] = helpers.get_course_image_url(course_data);
+        front_matter["course_description"] = course_data["description"];
+        front_matter["course_info"] = {};
+        front_matter["course_info"]["instructors"] = [];
```

Comment:
This can be

    instructors: course_data["instructors"].map(instructor => `Prof. ${instructor['first_name']} ${instructor['last_name']}`)